srv/modules/runners/populate.py

```diff
@@ -1075,15 +1068,15 @@ def engulf_existing_cluster(**kwargs):
     from . import validate
```

Comment:
Still not sure if that will make problems or how to solve that. I'm not seeing this in the utils directory.